test/e2e/components/vl-pill.js

```diff
@@ -57,4 +57,27 @@ class VlPill extends VlElement {
     }
 }
 
-module.exports = VlPill;
+class VlButtonPill extends VlElement {
```

Comment:
We kunnen denk ik beter een base class maken, de tegenhanger van VlPillElement, waar de e2e component objecten van kunnen extenden.